src-executables/Main-trident.hs

```diff
@@ -100,8 +100,7 @@ renderVersion :: String
 renderVersion =
     "trident v" ++ showVersion version ++ " for poseidon v" ++
     intercalate ", v" (map showPoseidonVersion validPoseidonVersions) ++ "\n" ++
-    "https://poseidon-framework.github.io" ++ "\n" ++
-    ")<(({°> ~ ────E ~ <°}))>("
+    "https://poseidon-framework.github.io"
```

Comment:
😂 I must have put you off with my questions about this ASCII art... sorry.

Reply:
Stephen also didn't understand it, and asked if something is wrong with the terminal output. So I guess then it's just confusing and should be scrapped. I'm not tooo sad about it :slightly_smiling_face:

Reply:
Haha... OK